pom.xml

```diff
@@ -45,7 +45,7 @@
 
     <properties>
         <jdk.version>1.8</jdk.version>
-        <jet.version>4.5</jet.version>
+        <platform.version>5.0-SNAPSHOT</platform.version>
```

Comment:
shall we remove the platform in the light now recent changes

Reply:
yes, it makes sense, I will rather use `hazelcast.version` for this property